speech/speech.py

```diff
@@ -104,15 +104,10 @@ def try_setup_api(self):
         try:
             voice_client = await channel.connect()
 
-            options = "-filter \"volume=volume=0.3\""
+            options = '-filter "volume=volume=0.3"'
 
             audio_source = discord.FFmpegPCMAudio(audio_path, options=options)
-            voice_client.play(audio_source)
-
-            while not audio_player.is_done():
-                await asyncio.sleep(0.01)
-
-            await voice_client.disconnect()
+            voice_client.play(audio_source, after = corowrap(voice_client.disconnect))
```

Comment:
Instead of doing the weird global RPADBOT thing, just have corowrap accept the bot as an argument.

I think there's a bug in corowrap though, it takes ctx as an argument and isn't used, and you don't pass it here.

Reply:
oh it shouldn't take ctx.  it was gonna use that to get the bot.  honestly all it needs is an event loop and it seems easier to just grab the one from rpadutils cog.  i could pass in bot, but i foresee using corowrap often